src/components/live-tag/live-tag.js

```diff
@@ -77,13 +77,14 @@ class LiveTag extends Component {
   /**
    * render live tag component
    *
-   * @param {*} props - component props
    * @returns {React$Element} component element
    * @memberof LiveTag
    */
-  render(props: any): React$Element<any> {
+  render(): React$Element<any> {
     const tagStyleClass = [style.liveTag];
-    if (props.isDvr && !this.isOnLiveEdge()) tagStyleClass.push(style.nonLivePlayhead);
```

Comment:
why `isDvr` removed? if needed so remove it from mapStateToProps as well

Reply:
If you are out-of-sync (paused for example) also on live without DVR, we would like to see indication on the live tag.

Reply:
this is new behavior. how is related to this change?

Reply:
Now the calculation is different so we can support it. Like before it's a bug.